app/src/main/kotlin/pl/gov/mc/protego/ui/registration/RegistrationViewModel.kt

```diff
@@ -17,21 +18,19 @@ class RegistrationViewModel(
     private val msisdnValidator: MsisdnValidator,
     private val protegoServer: ProtegoServer,
     private val session: Session
-)  : ViewModel() {
+) : ViewModel() {
 
-    val msisdnError = MutableLiveData<String?>()
+    val msisdnError = MutableLiveData<Int?>()
```

Comment:
Do powyższej sugsestii:
```suggestion
    val msisdnError = MutableLiveData<@StringRes Int?>()
```
Nie dam głowy czy będzie działać.

Reply:
Ok